microsite/src/main/docs/reference/source-generation.md

```diff
@@ -14,6 +14,8 @@ Scala source code from Avro/Protobuf/OpenAPI IDL files.
 
 This section explains each of the sbt plugin's settings.
 
+TODO: should I add a section explaining how this plugin is disabled by default, or is that more appropriate in the other settings?
```

Comment:
At this point, you can add a reminder that the sbt plugin needs to be enabled, so something like the note above would work too.

Reply:
works for me!